.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig is awesome:
+http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[*]
+insert_final_newline = true
```

Comment:
I recommend omitting this line, unless you want to see pull requests containing end-of-file newline changes that are completely unrelated to the topic of the pull request. For example, any time someone manually edits a project file it will add the newline. Then anytime someone edits the project file automatically (such as adding a new file within Visual Studio), it will remove it again.